quickfixj-core/src/main/java/quickfix/SessionSettings.java

```diff
@@ -86,13 +86,34 @@
     // problems with moving configuration files between *nix and Windows.
     private static final String NEWLINE = "\r\n";
 
-    private Properties variableValues = System.getProperties();
+    private Properties variableValues;
```

Comment:
This is still a terrible name for a variable IMO but we could fix that in another review. 

To be completely obtuse, it's a variable property that contains contains variable properties.

`modifiableValues` would move us in the right direction. I'll take that up in another PR when this is merged.